images/linux/scripts/installers/2004/php.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+################################################################################
+##  File:  php.sh
+##  Desc:  Installs php
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/os.sh
+
+set -e
+
+# Install PHP
+
+if isUbuntu20 ; then
```

Comment:
I think this condition is not needed because this file in Ubuntu 20 folder

Reply:
Agree